packages/config/config/devices/0x0270/bvs-zwu.json

```diff
@@ -0,0 +1,609 @@
+{
+	"manufacturer": "Custos",
+	"manufacturerId": "0x0270",
+	"label": "BVS-ZWU",
+	"description": "Water Ball Valve Servo",
+	"devices": [
+		{
+			"productType": "0x0101",
+			"productId": "0x000a",
+			"zwaveAllianceId": [3885, 3916]
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Water Valve",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Leak Sensor",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Overheat Alarm",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Freeze Alarm",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
```

Comment:
If this file is the same as the Zooz one - do we want to turn all parameters into `$import`s from the Zooz file?

Reply:
I'll do that later when I template the Zooz directory. I didn't have time to mess with it.